test/junit/scala/collection/IterableTest.scala

```diff
@@ -106,6 +106,7 @@ class IterableTest {
     Assert.assertEquals(baselist.reverse, checklist)
   }
 
+  @Test
```

Comment:
Was there any reason NOT to run this test?

Reply:
Absolutely not!